.github/workflows/continuous-integration.yml

```diff
@@ -22,50 +21,9 @@ jobs:
           - {glpi-version: "10.0.x", php-version: "8.0", db-image: "mysql:8.0"}
           - {glpi-version: "10.0.x", php-version: "8.1", db-image: "mariadb:10.2"}
           - {glpi-version: "10.0.x", php-version: "8.2", db-image: "mariadb:11.0"}
-    container:
-      image: "ghcr.io/glpi-project/githubactions-glpi:php-${{ matrix.php-version }}-glpi-${{ matrix.glpi-version }}"
-      options: >-
-        --volume ${{ github.workspace }}:/var/glpi/plugins:rw
-    services:
-      db:
-        image: "ghcr.io/glpi-project/githubactions-${{ matrix.db-image }}"
-        env:
-          MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
-          MYSQL_DATABASE: "glpi"
-        options: >-
-          --shm-size=1g
-    defaults:
-      run:
-        working-directory: "/var/glpi/plugins/oauthimap"
-    steps:
-      - name: "Checkout"
-        uses: "actions/checkout@v4"
-        with:
-            path: "oauthimap"
-      - name: "Get Composer cache directory"
-        id: "composer-cache"
-        run: |
-          echo "::set-output name=dir::$(composer config cache-files-dir)"
-      - name: "Restore dependencies cache"
-        uses: "actions/cache@v3"
-        with:
-          path: "${{ steps.composer-cache.outputs.dir }}"
-          key: "${{ github.job }}-${{ matrix.php-version }}-dependencies-${{ hashFiles('**/composer.lock') }}"
-      - name: "Install Composer dependencies"
-        run: |
-          composer install --ansi --no-interaction --no-progress --prefer-dist
-      - name: "PHP Parallel Lint"
-        run: |
-          vendor/bin/parallel-lint --colors --checkstyle --exclude ./vendor/ . | vendor/bin/cs2pr
-      - name: "PHP_CodeSniffer"
-        run: |
-          vendor/bin/phpcs -q --report=checkstyle | vendor/bin/cs2pr
```

Comment:
This one seems not running

Reply:
They are, but as output is piped to `cs2pr`, it is actually hard to know it. I will fix this.